saleor/dashboard/staff/filters.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import unicode_literals
+
+from django_filters import (
+    CharFilter, ChoiceFilter, FilterSet, ModelMultipleChoiceFilter,
+    OrderingFilter)
+from django.contrib.auth.models import Group
+from django.utils.translation import pgettext_lazy
+from django import forms
+
+from ...core.utils.filters import filter_by_customer, filter_by_location
+from ...userprofile.models import User
+
+
+SORT_BY_FIELDS = (
+    ('email', 'email'),
+    ('default_billing_address__first_name', 'name'),
+    ('default_billing_address__city', 'location'))
+
+SORT_BY_FIELDS_LABELS = {
+    'email': pgettext_lazy(
+        'Customer list sorting option', 'email'),
+    'default_billing_address__first_name': pgettext_lazy(
+        'Customer list sorting option', 'name'),
+    'default_billing_address__city': pgettext_lazy(
+        'Customer list sorting option', 'location')}
+
+IS_ACTIVE_CHOICES = (
+    ('1', pgettext_lazy('Is active filter choice', 'Active')),
+    ('0', pgettext_lazy('Is active filter choice', 'Not active')))
+
+
+class StaffFilter(FilterSet):
+    name_or_email = CharFilter(
+        label=pgettext_lazy('Staff list filter label', 'Name or email'),
+        method=filter_by_customer)
+    location = CharFilter(
+        label=pgettext_lazy('Staff list filter label', 'Location'),
+        method=filter_by_location)
+    groups = ModelMultipleChoiceFilter(
+        label=pgettext_lazy('Staff list filter label', 'Groups'),
+        name='groups',
+        queryset=Group.objects.all())
+    is_active = ChoiceFilter(
```

Comment:
Can we use `models.BooleanField` here? I think it's more obvious than returning '1' or '0' as `str`

Reply:
Unfortunately BooleanField does not have empty_label and we want a custom string 'All' if no option was chosen.